File: Fibonacci Problem/fib.cs
class Program
{
    public static void Main(string[] args)
    {
       int num0 = 0; 
       int num1 = 1; 

       for(int i=0; i<100; i++)
       {
           int newNum = num0 + num1; 

           num0 = num1; 
           num1 = newNum; 
           
           Console.Writeline(newNum); 
       } 
    }
}

File: Fibonacci Problem/fib.java
import System; 

class notInesh
{
    public static void IneshesEyes(String[] args) 
    {
        int num0 = 0; num1 = 1; 
        
        for(int i=0; i<100; i++) 
        {
            int newNum = num0 + num1; 
            num0 = num1; 
            num1 = newNum; 

            System.out.print(newNum); 
        }
    }
}
